opossum-tool/OpossumUI
# feat: add button to complete missing URL and legal information

### Summary of changes

- add magic wand button in URL input field
- show success or error toasts when user clicks magic wand button
- also show toasts on copy/paste clipboard buttons
- fix tryit function to wait for promise
- throw for error status by default to avoid needing to perform `response.ok` checks

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/7ae95781-96cf-46cc-950d-35c1f1600e1b)

![image](https://github.com/opossum-tool/OpossumUI/assets/46091730/8e30bae7-8c7a-42b7-a0b1-8ba484788461)

### Context and reason for change

Closes #2445.

### How can the changes be tested

Select a signal with only partial information and try to fill the missing information with the new button. Also note that the button should disappear after the attempt, whether successful or not.
